### ISSUE TITLE
Restore the Jenkins VMs

### ISSUE THREAD

**Issue by style95** [apache/openwhisk#4939]
## Environment details:

* Apache VMs.

## Steps to reproduce the issue:

* Jenkins CI is missing now. 

## Additional information you deem important:
Previously we used 3 Apache VMs to run Jenkins and corresponding CI tests.
But at some point, we lost control and it is no longer working now.

**Comment by style95** [comment]
I had access to all 3 VMs.
But it seems there are only 2 of them running.

```
144.76.63.94    openwhisk1  openwhisk-vm1-he-de.apache.org
5.9.20.92       openwhisk2  openwhisk-vm2-he-de.apache.org
144.76.27.183   openwhisk3  openwhisk-vm3-he-de.apache.org
```

`openwhisk-vm3-he-de.apache.org` is no longer accessible now.
I will create an infra ticket for it.

I am looking into this file to figure out what Jenkins machines were for.
https://github.com/apache/openwhisk/blob/master/Jenkinsfile

It takes the following steps:
1. Set up a docker registry on the first machine.
2. Set up a lean openwhisk deployment.
3. Run tests for the lean openwhisk.
4. Set up full version of openwhisk.
5. Run `Shoot one invoker test`.

I will restore this functionality.

**Comment by style95** [comment]
One more thing I want to do is I want to set up a pull-request builder for [openwhisk-website](https://github.com/apache/openwhisk-website) in one of this VM.

So that we can easily figure out the changes with the rendered website.

Also, I expect we can easily add documentation to the website with it.
I feel we need to add many documentations such as how to set up the development environment, how the internal works, etc to attract more developer advocates.
I think we can accumulate our knowledge into the site.

**Comment by style95** [comment]
* Infra ticket: https://issues.apache.org/jira/browse/INFRA-20669

**Comment by style95** [comment]
It seems there is no jenkins job for this.
So I will create a new one.
![image](https://user-images.githubusercontent.com/3447251/89873304-2e640a00-dbf5-11ea-9d3d-689e74b9b0da.png)

**Comment by dgrove-oss** [comment]
I thought Apache Infrastructure informed us they were planning to deprovision those servers as they were under-utilized.  (See thread on openwhisk-private on July 16).

**Comment by style95** [comment]
@dgrove-oss 
Yes. They have been under-utilized because there was no Jenkins job running and finally decommissioned.
But I suppose we need to run failover tests for sanity check?
I already asked to the infra guy about the enablement.
